src/client/app/reducers/readings.js

```diff
@@ -16,7 +16,12 @@ import * as readingsActions from '../actions/readings';
  * @type {State~Readings}
  */
 const defaultState = {
```

Comment:
Since we aren't showing more than one type of graph at a time, why do we need to fetch more than one type of readings at a time?